---
title: "Statistical Methods for Functional Genomics"
subtitle: "abc"
date: "June 23-July 10, 2020"
author:
- name: Vince Carey
  affiliation: Harvard University, Boston, MA
  
- name: Sean Davis
  affiliation: National Cancer Institute, National Institutes of Health, Bethesda, MD
  
- name: Keegan Korthauer
  affiliation: University of British Columbia, Vancouver, Canada
  
- name: Mike Love
  affiliation: University of North Carolina, Chapel Hill, Chapel Hill, NC

- name: Tomas Rube
  affiliation: University of California, Merced, Merced, CA

output: 
  BiocStyle::html_document:
    toc: FALSE
  BiocStyle::pdf_document:
    toc: FALSE
vignette: >
  %\VignetteIndexEntry{Syllabus, Statistical Methods for Functional Genomics}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

# Overview

Over the past decade, high-throughput assays have become pervasive in
biological research due to both rapid technological advances and
decreases in overall cost. To properly analyze the large data sets
generated by such assays and thus make meaningful biological
inferences, both experimental and computational biologists must
understand the fundamental statistical principles underlying analysis
methods. This course is designed to build competence in statistical
methods for analyzing high-throughput data in genomics and molecular
biology; in this way it is not a traditional "bioinformatics" course.

Instruction and hands-on activities will utilize the free [R
programming environment](https://r-project.org) and leverage the
[Bioconductor](https://bioconductor.org) software ecosystem to analyze
exemplar datasets and learn statistical concepts. Basic programming,
data management, and reproducible computational research topics are
also included. While some familiarity with R can be useful, the course
begins with a primer and review of R data structures and programming. 

## Prerequisites

# Learning Objectives

# Materials

# Course Topics

- The R programming language and Bioconductor
  - Introduction to the R environment
  - Basic linux/unix command-line
  - R data structures
  - Manipulating data in data frames and matrices
  - Plotting and data visualization
  - Distributions
  - Hypothesis testing
  - Exploratory data analysis
  - Genomic range operations
  - Bioconductor data structures
  - Accessing public data resources
- Statistical concepts
  - Statistical distributions
  - Hypothesis testing
  - Experimental design
  - P-values
  - Bayes theorem
  - Multiple testing correction
  - Dimensionality reduction
  - Weight matrices
  - Regression
  - Linear models
  - Introductory machine learning
- Biological domains
  - RNA-seq quantification, normalization, and differential expression
  - Gene set enrichment analysis
  - Expression QTLs
  - Allele specific expression
  - Single-cell transcriptomics
  - ATAC-seq
  - Transcription factor binding and motif discovery
  - DNA methylation
- Computation and programming
  - Linux/unix command line
  - Literate programming (R markdown)
  - Version control with git
  - Building an R package
  - Data management with R and Linux

# Logistics

## Computer Equipment

- Macintosh with 16GB RAM, 40GB of free disk space

## Software

Ability to install R packages as a user....

- R version 3.7
- Rstudio
- Bioconductor (TODO--version)

## Daily schedule

While the schedule will vary somewhat from day-to-day, a typical daily
routine looks like this.

| Approximate times | Activity  |
| --------------- | ------------------------------------------------ |
| 7:30-9:00 AM    |   Breakfast                                        |
| 9:00-12:00 Noon |   lecture/lab                                      |
| 12:00-1:00 PM   |   Lunch                                            |
| 1:00-4:00 PM    |   lecture/lab                                      |
| 4:00-5:00 PM    |   Occasional guest lecture or continue lecture/lab |
| 5:00-7:00 PM    |   Dinner                                           |
| 7:00-8:00 PM    |   Guest lecture (cont.) or lecture/lab             |
| 8:00-... PM     |   Open lab                                         |

# Guest Speakers (TODO)

- Brittany Adamson, Princeton University, Princeton, NJ
- Elana Fertig, Johns Hopkins University, Baltimore, MD
- Tuuli Lappalainen, New York Genome Center & Columbia University, New York, NY
- Karen Mohlke, University of North Carolina, Chapel Hill, NC
- Robert Patro, Stony Brook University, Stony Brook, NY

# Cold Spring Harbor Campus Life

# Course policies

- [Diversity and Inclusion](https://meetings.cshl.edu/policies.aspx?meet=BIOHIST&year=20#diversity)
- [Code of Conduct](https://meetings.cshl.edu/policies.aspx?meet=BIOHIST&year=20#conduct)

# Instructors

## Office hours

We do not maintain office hours but are available throughout the
course for discussion and advice (and to learn from the
students). Students are encouraged to work on their own datasets as
the course progresses, offering opportunities to talk through
approaches with other students and instructors on an _ad hoc_ basis.

## Vince Carey

- email: stvjc@channing.harvard.edu

## Sean Davis {-}

- email: seandavi@gmail.com
- home page: https://seandavi.github.io/
- twitter: [seandavis12](https://twitter.com/seandavis12)
- github: [seandavi](https://github.com/seandavi)

## Keegan Korthauer

- email: keegan@stat.ubc.ca

## Mike Love

- email: michaelisaiahlove@gmail.com

## Tomas Rube

- email: rube.tomas@gmail.com
